metadata/reportssql/testing_report_functions.sql

```diff
@@ -822,6 +822,157 @@ BEGIN
 END$$
 DELIMITER ;
 
+-- getPatientHIVTestDate
+
+DROP FUNCTION IF EXISTS getPatientHIVTestDate;
+
+DELIMITER $$
+CREATE FUNCTION getPatientHIVTestDate(
```

Comment:
@tmvumbi2 I think we should use getHIVTestDate here (and in future for other functions)